### DISCUSSION TITLE
怎么给 NDialogProvider 设置 z-index?

### DISCUSSION THREAD

**Discussion by wuchangming** [tusen-ai/naive-ui#4276] [Q&A]
现在只能这样设置
```
.n-modal-container {
    z-index: 9999 !important;
}
```
有正常的方式不？

**Comment by XieZongChen** [comment]
![image](https://user-images.githubusercontent.com/46394163/211453990-8cc254c2-5685-4925-b7ec-c6cbfbda0131.png)

**Reply by wuchangming** [reply]
这个也行，但就是不能统一给所有用到 provider 的都自定义设置一个
